tests/api_test/api_test.cpp

```diff
@@ -1334,3 +1334,44 @@ TEST_CASE("test_ringbuffer_wraparound", "[stress]")
     // Clean up.
     bpf_object__close(object);
 }
+
+TEST_CASE("Test program order", "[native_tests]")
+{
+    struct bpf_object* object = nullptr;
+    fd_t program_fd;
+    uint32_t program_count = 4;
+    int result;
+
+    REQUIRE(
+        _program_load_helper(
+            "multiple_programs.sys", BPF_PROG_TYPE_SAMPLE, EBPF_EXECUTION_NATIVE, &object, &program_fd) == 0);
+
+    // Get all the 4 programs in the native object, and invoke them using bpf_prog_test_run.
```

Comment:
```suggestion
    // Get all 4 programs in the native object, and invoke them using bpf_prog_test_run.
```
grammar suggestion: either remove "all" or "the".